assets/js/hooks/snackbar.js

```diff
@@ -0,0 +1,39 @@
+const queue = [];
+
+const isVisible = (el) => !el.classList.contains("hidden") && !el.classList.contains("animate-fadeout")
+
+const visibleEls = () => Array.from(document.getElementsByClassName('moon-snackbar')).filter(isVisible)
```

Comment:
Please don't use abbreviations for function names unless they are well defined industry standards. `El` and `Els` meaning here is maybe not ambiguous but it is still a good practice to be explicit, basically "don't-make-me-guess" principle.